[IMP] Fiscal localizations: Add info about Teledec and liasse fiscale (#11454)
odoo/documentation: content/applications/finance/fiscal_localizations/france.rst

- Compte de résultats
- Plan de Taxes France

.. _localization/france/liasse-fiscale:

Liasse fiscale
==============

The *liasse fiscale* (tax returns) is a collection of standardized financial documents that
businesses must submit annually to the tax authorities. It comprehensively summarizes the company’s

> financial activities and determines corporate taxes. This documentation allows tax authorities to
> review a company's financial situation consistently and transparently for taxation purposes.

Reviewer: ```suggestion
financial activities and determines corporate taxes.
```
I think we can safely remove this sentence as it's not really necessary IMO.

- Compte de résultats
- Plan de Taxes France

.. _localization/france/liasse-fiscale:

Liasse fiscale
==============

The "liasse fiscale" (tax returns) is a collection of standardized financial documents that
businesses must submit annually to the tax authorities. It comprehensively summarizes the company’s
financial activities and determines corporate taxes.
This documentation allows tax authorities to review a company's financial situation consistently and
transparently for taxation purposes. Preparing the liasse fiscale requires following French
accounting standards and is usually done with the help of accountants or tax professionals.

.. _localization/france/teledec:

Teledec
-------

Teledec is a tool to prepare and submit tax returns by utilizing data from accounting records. To
electronically send the company's liasse fiscale to the DGFiP (Direction Générale des Finances

> Publiques) and synchronize the information between Teledec and Odoo, follow these steps:

Author: @auva-odoo Yes, you're right 😁

the SIREN number of the company.

.. note::
   If the account has already been created, click :guilabel:`Se connecter` (Connect) at the top of
   the page.

.. _localization/france/teledec-registration:

Company registration and fiscal year information
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

To register the company in Teledec, go to :guilabel:`Vos entreprises` (Your companies) and click
:guilabel:`Enregistrer votre entreprise` (Register your company). Make sure to fill in the following
company information in the :guilabel:`Coordonnées de l'entreprise` (Company's details) and
:guilabel:`Représentant légal` (Legal representative) sections:

- :guilabel:`Nom de l'entreprise`: company's name
- :guilabel:`Forme juridique`: company's legal form to select
- :guilabel:`Les comptes sont clôturés le`: closing date
- :guilabel:`Régime d'imposition, choix de la liasse`: Tax scheme and tax return choice to select
- :guilabel:`Adresse du siège social`: Head office address
- :guilabel:`Nom du représentant légal`: Legal representative's name
- :guilabel:`Agissant en qualité de`: legal representative's function
- :guilabel:`Numéro de téléphone`: phone number

> Click :guilabel:`Sauvegarder` (Save) to open the following
> :guilabel:`Enregistrer les informations sur l'exercice` step. Then, fill in information on the

Reviewer: ```suggestion
Click :guilabel:`Sauvegarder` (Save) to display the next step
:guilabel:`Informations générales sur l'exercice déclaré` (General information about the declared fiscal year). Then, fill in information on the
```
(according to the screenshot in the Google doc? and I added a translation, not super sure about "declared")

financial year such as the fiscal year start and end dates or the closing date and duration of the
previous fiscal period.

> After saving, the list of documents included in the liasse fiscale will be accessible, including
> both standard tax forms and those customized for the company’s tax return.

Reviewer: ```suggestion
After saving, the list of documents included in the *liasse fiscale* is displayed, including
both standard tax forms and those customized for the company’s tax return.
```

.. note::
   - The :guilabel:`Etat` (Status) column shows the progress of the document filing.
   - To print a blank version of the declaration, click :guilabel:`Imprimer la déclaration` and
     select the :guilabel:`Imprimer la déclaration avec les notices` option.

> 

Reviewer: I would use a tip instead of a note and add a list item about how you can click "Compléter" to fill out a document (and then need to save). What do you think?

.. _localization/france/teledec-synchronization:

Odoo synchronization
~~~~~~~~~~~~~~~~~~~~

To enable Odoo to automatically fill in the data for the :guilabel:`Liasse fiscale`, click
:guilabel:`Autres actions` (Other actions) in the top-right corner and select
:guilabel:`Synchroniser avec un logiciel tiers` (Synchronize with third-party software), then
:guilabel:`Synchroniser cette liasse avec Odoo` (Synchronize this liasse with Odoo).

In the :guilabel:`Synchroniser cette liasse avec Odoo` window, fill in the following
information to complete the synchronization:

> - :guilabel:`Nom / URL complète de la base de données ODOO`: Odoo database name. If it is the full
>   url of the database, check the :guilabel:`Je voudrais donner une url complète hors .odoo.com`
>   option.

Reviewer: ```suggestion
- :guilabel:`Nom / URL complète de la base de données ODOO`: Odoo database name or URL. To provide the full URL of the database, enable :guilabel:`Je voudrais donner une url complète hors .odoo.com`.
```

- :guilabel:`Nom de l'utilisateur`: User name associated with the Odoo account.

> - :guilabel:`Clé API`: :ref:`API key <api/external_api/keys>` generated by Odoo instance.

Reviewer: ```suggestion
- :guilabel:`Clé API`: :ref:`API key <api/external_api/keys>` generated by the Odoo instance.
```

Next, click :guilabel:`Importer` to synch data from Odoo. In the
:guilabel:`Confirmation de la synchronisation de liasse avec Odoo` window, review the amounts and
make any necessary changes. Then click :guilabel:`Importer la balance` to confirm the
synchronization of the liasse fiscale with Odoo and import the balance.

.. important::

>    Some documents may be fully replaced after the import, even if they were updated beforehand.

Reviewer: I'm not sure I understand what this means?

Odoo absolves itself of all and any responsibility in case of changes
in the module’s functions caused by 3rd party applications not certified by Odoo.

.. _localization/france/anti-fraud-more-info:

> More Information

Author: @auva-odoo oupsie ;)
